packages/opentelemetry-exporter-zipkin/src/zipkin.ts

```diff
@@ -39,6 +39,7 @@ export class ZipkinExporter implements SpanExporter {
   private readonly _statusCodeTagName: string;
   private readonly _statusDescriptionTagName: string;
   private readonly _reqOpts: http.RequestOptions;
+  private _shutdown: boolean;
```

Comment:
I can suggest just changing name maybe to something like "isShuttingDown" or "willShutdown" as generally "shutdown" sounds like method not like boolean expression

Reply:
decided to go for `isShutdown`. It's more a state then a transition therefore I think it fits better then `isShuttingDown` or `willShutdown`.